0.3s is really too much for the cursor movement, it makes me vomit.
I'm sure that this insane default cannot suite anyone.

--- a/src/editor.hpp
+++ b/src/editor.hpp
@@ -444,7 +444,7 @@
   int animation_frame_interval_ms = 10;
   int scroll_animation_frame_interval = 10;
   float scroll_animation_time = 0.3f;
-  float cursor_animation_time = 0.3f;
+  float cursor_animation_time = 0.05f;
   int cursor_animation_frametime_ms = 10;
   bool hide_cursor_while_typing = false;
   Mouse mouse;
